typescript/packages/plugins/mode-governance/src/mode-governance.service.ts

```diff
@@ -0,0 +1,115 @@
+import { Tool } from "@goat-sdk/core";
+import { EVMWalletClient } from "@goat-sdk/wallet-evm";
+import { parseUnits, formatUnits } from "viem";
+import { VOTING_ESCROW_ABI, MODE_TOKEN_ABI, BPT_TOKEN_ABI } from "./abi";
+import {
+    MODE_TOKEN_ADDRESS,
+    BPT_TOKEN_ADDRESS,
+    MODE_VOTING_ESCROW,
+    BPT_VOTING_ESCROW,
+} from "./constants";
+import {
+    StakeParameters,
+    GetStakeInfoParameters,
+    GetBalanceParameters,
+} from "./parameters";
+
+export class ModeGovernanceService {
+    @Tool({
+        description: "Stakes MODE or BPT tokens in the Mode governance system",
```

Comment:
Descriptions should be imperative, "Stake MODE ..."

Reply:
sure